Dockerfile

```diff
@@ -29,9 +29,9 @@ COPY --chown=mongodb:mongodb . /data/db2/
 RUN mongod --fork --logpath /var/log/mongodb.log --dbpath /data/db2 \
   && npm run db:refresh \
   && mongod --dbpath /data/db2 --shutdown \
-  && chown -R mongodb /data/db2
+  && chown -R mongodb:mongodb /data/db2
```

Comment:
on the machine i tested with the docker build hangs on this run statement for a good long while (~120s) , succeeds just fine but takes some time